application/models/Survey.php

```diff
@@ -1630,13 +1635,11 @@ public function search()
                     $criteria->compare("t.active", 'Y');
                     $subCriteria1 = new CDbCriteria();
                     $subCriteria2 = new CDbCriteria();
-                    $subCriteria1->addCondition("'{$sNow}' > t.startdate", 'OR');
+                    $subCriteria1->addCondition("'{$sNow}' > t.startdate", 'AND');
```

Comment:
maybe can move to compare to use PDO (and then DB cache)?

Reply:
Use PDO? You mean use bind param?

Reply:
`$subCriteria1->compare("startdate","<{$sNow}");`